FakebookNotifications/FakebookNotifications.DataAccess/Models/Notification.cs

```diff
@@ -0,0 +1,33 @@
+﻿using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+
+namespace FakebookNotifications.DataAccess.Models
+{
+    public class Notification
+    {
+        //mongo object id
+        [BsonId]
+        [BsonRepresentation(BsonType.ObjectId)]
+        public string Id { get; set; }
+
+        //notification type: post/comment/tag
+        [BsonElement("type")]
+        public string Type { get; set; }
+
+        //user logged in to receive notifications
+        [BsonElement("user")]
+        public User LoggedInUser { get; set; }
```

Comment:
don't put circular references in your mongodb object model without a pressing need. this could be just a user ID.
same with the triggeruser below.